example/ibc/main.py

```diff
@@ -1,202 +0,0 @@
-import json
-import subprocess
-import time
-from pathlib import Path
-
-import requests
-import yaml
-from pystarport.cluster import (ClusterCLI, find_account, init_cluster,
-                                interact, start_cluster)
-from pystarport.ports import api_port
-
-from chainlibpy import Transaction, Wallet
-from chainlibpy.amino import Coin, StdFee, TimeoutHeight
-from chainlibpy.amino.message import IbcMsgTransfer
-
```

Comment:
the ibc example can be rewritten, we can create another issue maybe and add a `TODO` comment here.

Reply:
TODO https://github.com/crypto-org-chain/chainlibpy/issues/22